dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyTreeFormatter.java

```diff
@@ -98,14 +98,15 @@ private static void formatDependencyPathTree(
   }
 
   /**
-   * Builds ListMultiMap that represents a Maven dependency tree of parent-children relationship.
-   * The root node is represented as an empty {@link DependencyPath} and its children are
-   * the values for the root node.
+   * Builds ListMultimap that represents a Maven dependency tree of parent-children relationship.
    *
    * @param dependencyPaths dependency path instances without assuming any order
-   * @return ListMultiMap representing a Maven dependency tree
+   * @return ListMultimap representing a Maven dependency tree of parent-children relationship. The
+   *     {@link DependencyPath} representing the root Maven artifact is available by querying the
+   *     tree with an empty {@link DependencyPath} instance. For a {@code node} in the returned
+   *     {@code tree}, its children are retrieved by {@code tree.get(node)}.
    */
-  private static ListMultimap<DependencyPath, DependencyPath> buildDependencyPathTree(
+  public static ListMultimap<DependencyPath, DependencyPath> buildDependencyPathTree(
```

Comment:
We need a ListMultimap for order, but do we need to expose this in the API? Should this method simply return a Multimap instead?